src/psyclone/tests/dependency_test.py

```diff
@@ -358,18 +361,18 @@ def test_lfric_cma(fortran_writer):
     _, invoke_info = get_invoke("20.0_cma_assembly.f90", "lfric", idx=0)
     invoke_info.setup_psy_layer_symbols()
     var_info = str(invoke_info.schedule.reference_accesses())
-    assert "ncell_2d: READ" in var_info
-    assert "cma_op1_alpha: READ" in var_info
-    assert "cma_op1_bandwidth: READ" in var_info
-    assert "cma_op1_beta: READ" in var_info
-    assert "cma_op1_gamma_m: READ" in var_info
-    assert "cma_op1_gamma_p: READ" in var_info
+    assert "ncell_2d: WRITE+READ" in var_info
```

Comment:
Any idea why this is now "WRITE+READ" instead of "READ"? Is it because it's passed as an argument to a kernel?

Reply:
Aha. Thanks very much.